app/models/user.rb

```diff
@@ -20,5 +20,7 @@ class User < ApplicationRecord
 
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable,
-         :confirmable, :omniauthable
+         :confirmable
+
+  devise :omniauthable, omniauth_providers: %i[github stackexchange]
```

Comment:
This was important for being able to omniauth more than one service, but I have to wonder if there is a cleaner way to handle it.